src/main/java/com/pinterest/secor/main/OffsetCheckMain.java

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
I'm wondering if we couldn't make this part of the ZookeeperClient? Conceptually it fits as it fixes data in ZK. What do you think?